### PR TITLE
docs: update params of get_live_cell

### PR DESCRIPTION
**Pull request by Keith-CY**
update params from `cell_out_point` to `out_point`

### CODE REVIEW

**Comment by doitian** [comment]
This markdown file is generated via `devtools/doc/rpc.py`. Please update `rpc.json` and generate the markdown file.

**Comment by keroro520** [comment]
Please update `rpc/json/rpc.json`, and manually update README by run `python devtools/doc/rpc.py json/rpc.json > rpc/README.md`.

**Comment by Keith-CY** [comment]
> This markdown file is generated via `devtools/doc/rpc.py`. Please update `rpc.json` and generate the markdown file.

the `rpc.json` is correct while the `README` is incorrect

**Comment by keroro520** [comment]
> > This markdown file is generated via `devtools/doc/rpc.py`. Please update `rpc.json` and generate the markdown file.
> 
> the `rpc.json` is correct while the `README` is incorrect

Somebody forgot to generate document. Thanks for your help to correct it :D

File: rpc/README.md [hunk 1]
```diff
@@ -559,7 +559,7 @@ Returns the information about a cell by out_point.
 
 #### Parameters
 
-    out_point - OutPoint object {"tx_hash": <tx_hash>, "index": <index>}.
+    out_point - OutPoint object {"block_hash": "0xa5224507f309ea8e98b5af50cb21e9e957845c6faef730971292df974f6a4e23", "cell": {"tx_hash": <tx_hash>, "index": <index>}}.
```
Comment (keroro520) [inline rpc/README.md]: Here is type description. So I prefer to use `<block_hash>` as placeholder, but not a specific hash string.

Comment (keroro520) [reply rpc/README.md]: I will correct it in https://github.com/nervosnetwork/ckb/pull/762

**Comment by doitian** [comment]
> > > This markdown file is generated via `devtools/doc/rpc.py`. Please update `rpc.json` and generate the markdown file.
> > 
> > 
> > the `rpc.json` is correct while the `README` is incorrect
> 
> Somebody forgot to generate document. Thanks for your help to correct it :D

Could you add a ci job to generate the RPC doc and then check whether it is dirty? Remeber to add it in both `make ci` and `devtools/ci/script.sh`

**Pull request closed without merging**